.azure-pipelines/official-build-cache.yml

```diff
@@ -0,0 +1,40 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 */6 * * *"
+  displayName: Build cache quickly
+  branches:
+    include:
+    - master
+
+trigger: none
+pr: none
+
+stages:
+- stage: Build
+  pool: sonicbld
+  variables:
+    CACHE_MODE: cache
+  jobs:
+  - template: azure-pipelines-build.yml
+    parameters:
+      buildOptions: 'USERNAME=admin SONIC_BUILD_JOBS=$(nproc) ${{ variables.VERSION_CONTROL_OPTIONS }}'
+      jobGroups:
+        - name: vs
+          variables:
+            dbg_image: yes
+        - name: broadcom
+          variables:
+            dbg_image: yes
+            swi_image: yes
+            raw_image: yes
+            sync_rpc_image: yes
+            platform_rpc: brcm
+        - name: mellanox
+          variables:
+            dbg_image: yes
+            sync_rpc_image: yes
```

Comment:
The option has changed in master/202012 branch, change to docker_syncd_rpc_image, see https://github.com/Azure/sonic-buildimage/blob/master/.azure-pipelines/azure-pipelines-build.yml